lib/cb/clients/recommendation.rb

```diff
@@ -65,11 +65,10 @@ def self.for_user(*args)
       def self.for_company(company_did)
         my_api = Cb::Utils::Api.instance
         json_hash = my_api.cb_get(Cb.configuration.uri_recommendation_for_company, query: { CompanyDID: company_did })
-
         jobs = []
         if json_hash.key?('Results')
           if json_hash['Results'].key?('JobRecommendation')
-            json_hash['Results']['JobRecommendation']['Jobs'].each do |cur_job|
+            json_hash['Results']['JobRecommendation']['Jobs']['CompanyJob'].each do |cur_job|
```

Comment:
So, if you look at line 71/70, there's a nil check for `"JobRecommendation"`. You should probably do that for `"Jobs"` inside of it as well to avoid calling a method `:[]` on a `NilClass`.

You could change `if json_hash['Results'].key?('JobRecommendation')` to look like 
`if json_hash['Results'].key?('JobRecommendation') && json_hash['Results']['JobRecommendation'].key?('Jobs')`